docs/data/charts/lines/ConnectNulls.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import Stack from '@mui/material/Stack';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import Checkbox from '@mui/material/Checkbox';
+import { LineChart } from '@mui/x-charts/LineChart';
+
+export default function ConnectNulls() {
```

Comment:
Nitpick: consider going with different data points so that the lines would not overlap or change the color palette.
Currently, this example looks "buggy" in dark mode. 🙈 
![Screenshot 2023-11-01 at 17 10 04](https://github.com/mui/mui-x/assets/4941090/b8ac1b49-36b3-4bf0-b796-68faff630661)

Reply:
Effectively, this is better

![image](https://github.com/mui/mui-x/assets/45398769/ad0b794b-cf6e-4a3e-a2d8-34024afda236)